src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -160,4 +176,19 @@ public void onShowReplayDetail() {
   public void onWatchButtonClicked() {
     replayService.runReplay(replay);
   }
+
+  public void onDeleteButtonClicked() {
+    notificationService.addNotification(new ImmediateNotification(
+      String.format("%s %s",
+          i18n.get("replay.deleteNotification.heading"),
+          replay.getTitle()),
```

Comment:
Rather than use string.format you should add the argument to the string in the messages.properties file with the placeholder {0} then pass the replay title as an argument to the i18n.get call. This avoids the inherent locale dependency of String.format
```suggestion
          i18n.get("replay.deleteNotification.heading", replay.getTitle())
```

Reply:
Nice, didn't realize this was a thing